scenarios/azure/aks/run.go

```diff
@@ -13,11 +23,82 @@ func Run(ctx *pulumi.Context) error {
 		return err
 	}
 
-	_, kubeConfig, err := aks.NewCluster(env, "aks", nil)
+	cluster, kubeConfig, err := aks.NewCluster(env, "aks", nil)
 	if err != nil {
 		return err
 	}
 
+	// Export cluster’s properties
 	ctx.Export("kubeconfig", kubeConfig)
+	ctx.Export("kube-cluster-name", cluster.Name.ApplyT(func(value string) (string, error) { return value, nil }))
```

Comment:
Why do we need an `ApplyT` here ?
The function looks like identity.

```suggestion
	ctx.Export("kube-cluster-name", cluster.Name)
```